src/FeedbackBot/Controllers/HomeController.cs

```diff
@@ -11,14 +11,28 @@ namespace FeedbackBot.Controllers
 {
     [Authorize(ActiveAuthenticationSchemes = "ucdcas")]
     public class HomeController : Controller
-    {  
-        public async Task<IActionResult> Index()
+    {
+
+        public static class Globals
+        {
+            public static String kerberos { get; set; } 
```

Comment:
@cszhu since I believe `User.Identity.Name` is stable throughout the lifecycle of this class, you could just set a local non-static string variable in the HomeController constructor that contains the kerberos of the current user.